snuba/cli/bulk_load.py

```diff
@@ -51,7 +51,11 @@ def bulk_load(
     )
     # TODO: see whether we need to pass options to the writer
     writer = BufferedWriterWrapper(
-        table_writer.get_bulk_writer(table_name=dest_table),
```

Comment:
I missed passing the `metrics` in https://github.com/getsentry/snuba/pull/1162, this has been broken for a while. 😅